### DISCUSSION TITLE
Welcome to Killer Discussions!

### DISCUSSION THREAD

**Discussion by ntaraujo** [ntaraujo/killer#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chipsoftwaretest** [comment]
After installing Killer my machine was infected with Wacapew!C
What is going on with your code?

**Reply by ntaraujo** [reply]
It is known that PyInstaller can produce .exes that are misinterpreted by antiviruses:
[example 1](https://stackoverflow.com/questions/43777106/program-made-with-pyinstaller-now-seen-as-a-trojan-horse-by-avg)
[example 2](https://www.reddit.com/r/Python/comments/9ri81s/my_pyinstallercompiled_exe_progs_are_victims_of/)
Sorry for the inconvenience. I'm sure your pc was not actually corrupted by this program, but yes, it is a high level problem to be solved here.
Next versions will include tries to fix. Take a look into the release notes when they come out.

**Reply by ntaraujo** [reply]
Hopefully fixed with [v0.7.3](https://github.com/ntaraujo/killer/releases/tag/v0.7.3)
If you still experience this problem, please open an issue or comment in #2
Have a nice day 🙂

**Reply by chipsoftwaretest** [reply]
Hi ntaraujo,
thank you for the swift response, the issue seems to be resolved with v0.7.3.
I did not mean to be rude but it was very unexpected behaviour especially after no engine detected it in VirusTotal. 

As of now I don't see the big advantage of Killer over regular Task Manager, or can you elaborate more?
I do however enjoy the star wars meme reference :)

**Reply by ntaraujo** [reply]
Hi again! Its okay, a virus is not a cool thing to have on your computer and I would be worried too.

About the advantages, Killer is not supposed to replace any other complete Task Manager or even the built-in one.
The advantages are not over other Task Managers but over them in this specific need (as the name say haha):  kill multiple processes at once.

Even I use Process Hacker when talking about advanced needs, but to kill chrome (without hotkeys) i would have to:
* select the search bar
* search chrome
* select the first of multiple process
* sometimes scroll to the end
* press `Shift`
* select the last process in list
* ensure I didn't selected any process not related to the search, as ProcessHacker add it to the list if the process was created after the search
* right click
* select `Terminate` or `Terminate tree`
* confirm in a pop-up that yes I want to terminate these processes

In Killer:
* search chrome (search bar is initially selected)
* check the general checkbox
* press `KILL` or `THE CHILDREN TOO`

Ofc you can use hotkeys in most Task Managers, but these are on plans for Killer's future versions too.

A complete Task Manager is if you want deep analysis and deep options. Some times we just want to discover who is the bad boy of memory consumption and... bump it off

Oh, we have a dark theme too 🌙

**Reply by chipsoftwaretest** [reply]
Thanks for the input. May I suggest a "noob mode" or something for users who are not as experienced in distinguishing between system processes and their software? 

Also in actual Task Manager "Killer" is seen as "That Damn Chrome" lol

**Comment by elemen-flut** [comment]
![image](https://user-images.githubusercontent.com/73562667/148820196-f8ef091e-df62-41fc-9ac0-da79a4b67d65.png)

How to solve please help me

**Reply by ntaraujo** [reply]
[https://stackoverflow.com/questions/64655079/cannot-import-from-pywinauto-importerror-dll-load-failed-while-importing-win32](https://stackoverflow.com/questions/64655079/cannot-import-from-pywinauto-importerror-dll-load-failed-while-importing-win32)

Try installing the specific version of win32ui or a lower python version as 3.8 or 3.7

**Reply by elemen-flut** [reply]
![image](https://user-images.githubusercontent.com/73562667/148826584-15fa02f8-8d04-4033-ac9e-c0151fb14e3d.png)
Thanks god application ahs been runed but during application this error 
so how solve this error

**Comment by elemen-flut** [comment]
![image](https://user-images.githubusercontent.com/73562667/148827369-292dbb0b-3067-41f3-9ebf-faaaa1981bdc.png)
I will move process name then this error show

**Reply by ntaraujo** [reply]
does the program keep going? Or it freezes/doesn't work properly?

**Reply by elemen-flut** [reply]
> does the program keep going? Or it freezes/doesn't work properly?

yes sir program keep going but this error i can't understand 
and sir program is keep going but i will move button "process name" so this error can be genrated and show

**Reply by ntaraujo** [reply]
Windows will not retrieve all informations about the processes sometimes. This behavior is expected, somehow.

The exceptions occur in a separated thread, so the main program should not be affected.

If it is, like: you pressed the button to order by "Process Name" but it doesn't, then we have a bigger problem.

Otherwise, it doesn't matter so much. a series of error handling can be made, but it would probably slow down the app.

So, it was ordered by "Process Name" when you pressed the button?

**Reply by elemen-flut** [reply]
yes sir i will press the process name button then this error show otherwise all app perfectly run but i will press process name button then show this error

**Reply by ntaraujo** [reply]
So don't worry, it's expected.

I will create an issue to see if we can do a better error handling not compromising the speed, but I do not have time right now

**Reply by elemen-flut** [reply]
okay sir no issue and thankyou sir!
